CMake/CMakeLibs.cmake

```diff
@@ -28,8 +28,10 @@ macro(ADD_OSQUERY_TEST IS_CORE TEST_NAME SOURCE)
 endmacro(ADD_OSQUERY_TEST)
 
 macro(ADD_OSQUERY_PYTHON_TEST TEST_NAME SOURCE)
-  SYMLINK_FILE(${CMAKE_CURRENT_LIST_DIR}/${SOURCE} ${CMAKE_BINARY_DIR}/osquery/python_tests/${SOURCE})
-  add_test(NAME python_${TEST_NAME} COMMAND python ${CMAKE_BINARY_DIR}/osquery/python_tests/${SOURCE})
+  #SYMLINK_FILE(${CMAKE_CURRENT_LIST_DIR}/${SOURCE} ${CMAKE_BINARY_DIR}/osquery/python_tests/${SOURCE})
```

Comment:
Might as well remove this (and the macro for it) if we're going to run the tests out of the source dir.